### ISSUE TITLE
onEditorKeyDown method does not work for non ascii chars

### ISSUE THREAD

**Issue by moteus** [pkulchenko/ZeroBraneStudio#856]
I have this method in my package
```Lua
Package.onEditorKeyDown = function(self, editor, event)
    local key = event:GetKeyCode()
    local mod = event:GetModifiers()
    ide:Print('key is:', mod, key)
end
```

It invokes for all non characters keys and on keys in US layout.
But when I switch to Cyrlic layout this event does not fire for any character.

**Comment by pkulchenko** [comment]
I don't see this. I get the following for `abcморе`:

```
key is:	0	65
key is:	0	66
key is:	0	67
key is:	0	86
key is:	0	74
key is:	0	72
key is:	0	84
```

This is in line with what I'd expect. What OS are you on?

**Comment by moteus** [comment]
Ubuntu 16.04
And I use UTF8

**Comment by pkulchenko** [comment]
I can confirm this issue on Ubuntu, but it doesn't seem to be anything in the IDE that would cause this. I'll check if it's reproducible in wxwidgets master and open a ticket there if it is.

**Comment by pkulchenko** [comment]
I opened [wxiwdgets ticket](https://trac.wxwidgets.org/ticket/18054) to track.

**Comment by pkulchenko** [comment]
@moteus, I expect this to be resolved in #999; I'll be pushing Linux binaries there shortly.

**Comment by pkulchenko** [comment]
@moteus, this should be fixed in the available release beta branches; for example: https://github.com/pkulchenko/ZeroBraneStudio/tree/upgrade-binaries-190d. Could you verify?

**Comment by moteus** [comment]
I really do mnot remember why I want this.
But I try tou check it

**Comment by pkulchenko** [comment]
This has been implemented as part of the changes in #999 that have been merged.
